<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDlg</class>
 <widget class="QDialog" name="RegisterDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>11</y>
     <width>731</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadReg">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReg">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEixt">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Sans Serif&quot;;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
